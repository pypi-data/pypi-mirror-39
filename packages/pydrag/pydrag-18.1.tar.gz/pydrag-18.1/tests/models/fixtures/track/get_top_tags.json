{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&artist=AC+%2F+DC&autocorrect=1&format=json&method=track.getTopTags&track=Hells+Bell"
            },
            "response": {
                "body": {
                    "string": "{\"toptags\":{\"tag\":[{\"count\":100,\"name\":\"hard rock\",\"url\":\"https://www.last.fm/tag/hard+rock\"},{\"count\":59,\"name\":\"rock\",\"url\":\"https://www.last.fm/tag/rock\"},{\"count\":54,\"name\":\"classic rock\",\"url\":\"https://www.last.fm/tag/classic+rock\"},{\"count\":28,\"name\":\"ACDC\",\"url\":\"https://www.last.fm/tag/ACDC\"},{\"count\":25,\"name\":\"heavy metal\",\"url\":\"https://www.last.fm/tag/heavy+metal\"},{\"count\":14,\"name\":\"80s\",\"url\":\"https://www.last.fm/tag/80s\"},{\"count\":12,\"name\":\"AC/DC\",\"url\":\"https://www.last.fm/tag/AC%2FDC\"},{\"count\":11,\"name\":\"metal\",\"url\":\"https://www.last.fm/tag/metal\"},{\"count\":6,\"name\":\"australian\",\"url\":\"https://www.last.fm/tag/australian\"},{\"count\":6,\"name\":\"guitar\",\"url\":\"https://www.last.fm/tag/guitar\"},{\"count\":5,\"name\":\"hells bells\",\"url\":\"https://www.last.fm/tag/hells+bells\"},{\"count\":4,\"name\":\"Rock and Roll\",\"url\":\"https://www.last.fm/tag/Rock+and+Roll\"},{\"count\":4,\"name\":\"1980\",\"url\":\"https://www.last.fm/tag/1980\"},{\"count\":3,\"name\":\"70s\",\"url\":\"https://www.last.fm/tag/70s\"},{\"count\":3,\"name\":\"Supernatural\",\"url\":\"https://www.last.fm/tag/Supernatural\"},{\"count\":3,\"name\":\"heavy rock\",\"url\":\"https://www.last.fm/tag/heavy+rock\"},{\"count\":3,\"name\":\"favorites\",\"url\":\"https://www.last.fm/tag/favorites\"},{\"count\":3,\"name\":\"classic metal\",\"url\":\"https://www.last.fm/tag/classic+metal\"},{\"count\":2,\"name\":\"rock n roll\",\"url\":\"https://www.last.fm/tag/rock+n+roll\"},{\"count\":2,\"name\":\"Awesome\",\"url\":\"https://www.last.fm/tag/Awesome\"},{\"count\":2,\"name\":\"seen live\",\"url\":\"https://www.last.fm/tag/seen+live\"},{\"count\":2,\"name\":\"australia\",\"url\":\"https://www.last.fm/tag/australia\"},{\"count\":2,\"name\":\"live\",\"url\":\"https://www.last.fm/tag/live\"},{\"count\":2,\"name\":\"ac dc\",\"url\":\"https://www.last.fm/tag/ac+dc\"},{\"count\":2,\"name\":\"classic\",\"url\":\"https://www.last.fm/tag/classic\"},{\"count\":2,\"name\":\"Hard\",\"url\":\"https://www.last.fm/tag/Hard\"},{\"count\":2,\"name\":\"10 of 10 stars\",\"url\":\"https://www.last.fm/tag/10+of+10+stars\"},{\"count\":2,\"name\":\"hardrock\",\"url\":\"https://www.last.fm/tag/hardrock\"},{\"count\":2,\"name\":\"Ac-Dc\",\"url\":\"https://www.last.fm/tag/Ac-Dc\"},{\"count\":2,\"name\":\"alternative\",\"url\":\"https://www.last.fm/tag/alternative\"},{\"count\":1,\"name\":\"heavy\",\"url\":\"https://www.last.fm/tag/heavy\"},{\"count\":1,\"name\":\"bells\",\"url\":\"https://www.last.fm/tag/bells\"},{\"count\":1,\"name\":\"blues rock\",\"url\":\"https://www.last.fm/tag/blues+rock\"},{\"count\":1,\"name\":\"male vocalist\",\"url\":\"https://www.last.fm/tag/male+vocalist\"},{\"count\":1,\"name\":\"back in black\",\"url\":\"https://www.last.fm/tag/back+in+black\"}],\"@attr\":{\"artist\":\"AC/DC\",\"track\":\"Hells Bells\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "2574"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 16:15:46 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
