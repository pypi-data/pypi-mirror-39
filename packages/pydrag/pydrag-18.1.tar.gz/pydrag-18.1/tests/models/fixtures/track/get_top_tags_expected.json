{
    "artist": "AC/DC",
    "data": [
        {
            "count": 100,
            "name": "hard rock",
            "url": "https://www.last.fm/tag/hard+rock"
        },
        {
            "count": 59,
            "name": "rock",
            "url": "https://www.last.fm/tag/rock"
        },
        {
            "count": 54,
            "name": "classic rock",
            "url": "https://www.last.fm/tag/classic+rock"
        },
        {
            "count": 28,
            "name": "ACDC",
            "url": "https://www.last.fm/tag/ACDC"
        },
        {
            "count": 25,
            "name": "heavy metal",
            "url": "https://www.last.fm/tag/heavy+metal"
        },
        {
            "count": 14,
            "name": "80s",
            "url": "https://www.last.fm/tag/80s"
        },
        {
            "count": 12,
            "name": "AC/DC",
            "url": "https://www.last.fm/tag/AC%2FDC"
        },
        {
            "count": 11,
            "name": "metal",
            "url": "https://www.last.fm/tag/metal"
        },
        {
            "count": 6,
            "name": "australian",
            "url": "https://www.last.fm/tag/australian"
        },
        {
            "count": 6,
            "name": "guitar",
            "url": "https://www.last.fm/tag/guitar"
        },
        {
            "count": 5,
            "name": "hells bells",
            "url": "https://www.last.fm/tag/hells+bells"
        },
        {
            "count": 4,
            "name": "Rock and Roll",
            "url": "https://www.last.fm/tag/Rock+and+Roll"
        },
        {
            "count": 4,
            "name": "1980",
            "url": "https://www.last.fm/tag/1980"
        },
        {
            "count": 3,
            "name": "70s",
            "url": "https://www.last.fm/tag/70s"
        },
        {
            "count": 3,
            "name": "Supernatural",
            "url": "https://www.last.fm/tag/Supernatural"
        },
        {
            "count": 3,
            "name": "heavy rock",
            "url": "https://www.last.fm/tag/heavy+rock"
        },
        {
            "count": 3,
            "name": "favorites",
            "url": "https://www.last.fm/tag/favorites"
        },
        {
            "count": 3,
            "name": "classic metal",
            "url": "https://www.last.fm/tag/classic+metal"
        },
        {
            "count": 2,
            "name": "rock n roll",
            "url": "https://www.last.fm/tag/rock+n+roll"
        },
        {
            "count": 2,
            "name": "Awesome",
            "url": "https://www.last.fm/tag/Awesome"
        },
        {
            "count": 2,
            "name": "seen live",
            "url": "https://www.last.fm/tag/seen+live"
        },
        {
            "count": 2,
            "name": "australia",
            "url": "https://www.last.fm/tag/australia"
        },
        {
            "count": 2,
            "name": "live",
            "url": "https://www.last.fm/tag/live"
        },
        {
            "count": 2,
            "name": "ac dc",
            "url": "https://www.last.fm/tag/ac+dc"
        },
        {
            "count": 2,
            "name": "classic",
            "url": "https://www.last.fm/tag/classic"
        },
        {
            "count": 2,
            "name": "Hard",
            "url": "https://www.last.fm/tag/Hard"
        },
        {
            "count": 2,
            "name": "10 of 10 stars",
            "url": "https://www.last.fm/tag/10+of+10+stars"
        },
        {
            "count": 2,
            "name": "hardrock",
            "url": "https://www.last.fm/tag/hardrock"
        },
        {
            "count": 2,
            "name": "Ac-Dc",
            "url": "https://www.last.fm/tag/Ac-Dc"
        },
        {
            "count": 2,
            "name": "alternative",
            "url": "https://www.last.fm/tag/alternative"
        },
        {
            "count": 1,
            "name": "heavy",
            "url": "https://www.last.fm/tag/heavy"
        },
        {
            "count": 1,
            "name": "bells",
            "url": "https://www.last.fm/tag/bells"
        },
        {
            "count": 1,
            "name": "blues rock",
            "url": "https://www.last.fm/tag/blues+rock"
        },
        {
            "count": 1,
            "name": "male vocalist",
            "url": "https://www.last.fm/tag/male+vocalist"
        },
        {
            "count": 1,
            "name": "back in black",
            "url": "https://www.last.fm/tag/back+in+black"
        }
    ],
    "track": "Hells Bells"
}
